telemetry/ratelimiter.go

```diff
@@ -99,7 +104,7 @@ func (rl *RateLimiter) cleanup(ctx context.Context, cleanupEvery time.Duration)
 			return
 		case now := <-t.C:
 			numCleaned := 0
-			limiters := rl.values()
+			limiters := rl.valuesCopy()
```

Comment:
@richard-ramos I saw the map read and write concurrency error again, so trying another approach:D